src/main/kotlin/org/jetBrains/research/tasktracker/tracking/DefaultContentProvider.kt

```diff
@@ -54,13 +54,25 @@ object DefaultContentProvider {
         else -> ""
     }
 
-    private fun getPackage(extension: Extension): String {
-        val currentPackage =
-            "package $PLUGIN_NAME.${extension.name.lowercase(Locale.getDefault())}"
-        return when (extension) {
-            Extension.JAVA -> "$currentPackage;$lineSeparator$lineSeparator"
-            Extension.KOTLIN -> "$currentPackage$lineSeparator$lineSeparator"
+    private fun getPackage(task: Task): String {
+        val pack = "package $PLUGIN_NAME.${relativePathToPackage(task) ?: lowercaseExtension(task)}"
```

Comment:
It is better to use "currentPackage" or just "package" here for the variable name

Reply:
I'd suggest using `extension` as a primary folder for the task since we need to put tasks according to their languages to make source roots. E.g.:
```kotlin
"package $PLUGIN_NAME.${owercaseExtension(task)}.{relativePathToPackage(task)}"
```
BUT don't forget to handle the `null` case with `relativePathToPackage`